### DISCUSSION TITLE
Requesting votes from all DS team members!

### DISCUSSION THREAD

**Discussion by kelenelee** [hackforla/design-systems#417] [Polls]
If it's somewhere else, could you comment it below?

**Comment by allthatyazz** [comment]
All 3. 
For me personally, It depends on which platform I am in at the moment because I know all the possible places I can find the link.

**Reply by kelenelee** [reply]
Thanks for your comment :) if only these polls could be multi-select but they cannot

**Comment by hdchangie** [comment]
All three, but I will try to find it in Slack first, if that doesn't work, I will go to my calendar.

**Reply by kelenelee** [reply]
Good point – it looks like it really depends. I've reworded the poll question a bit
